x-pack/plugins/cases/server/client/cases/mock.ts

```diff
@@ -231,8 +231,10 @@ export const userActions: CaseUserActionsResponse = [
       username: 'elastic',
     },
     new_value:
-      '{"title":"Case SIR","tags":["sir"],"description":"testing sir","connector":{"id":"456","name":"ServiceNow SN","type":".servicenow-sir","fields":{"category":"Denial of Service","destIp":true,"malwareHash":true,"malwareUrl":true,"priority":"2","sourceIp":true,"subcategory":"45"}},"settings":{"syncAlerts":true}}',
+      '{"title":"Case SIR","tags":["sir"],"description":"testing sir","connector":{"name":"ServiceNow SN","type":".servicenow-sir","fields":{"category":"Denial of Service","destIp":true,"malwareHash":true,"malwareUrl":true,"priority":"2","sourceIp":true,"subcategory":"45"}},"settings":{"syncAlerts":true}}',
```

Comment:
The changes in this file are removing `connector.id` and adding the `*_val_connector_id` fields.